query-node/mappings/src/content/nft.ts

```diff
@@ -1012,7 +1048,12 @@ export async function contentNft_NftBought({ event, store }: EventContext & Stor
   // specific event processing
 
   // load video
-  const video = await getRequiredExistingEntity(store, Video, videoId.toString(), 'Non-existing video was bought')
+  const video = await getRequiredExistingEntity(store, Video, videoId.toString(), 'Non-existing video was bought', [
+    'nft',
+    'nft.ownerMember',
+    'nft.ownerCuratorGroup',
+  ])
+  const { nft } = video
```

Comment:
`getNftFromVideo` can be used here the same way as described here https://github.com/Joystream/joystream/pull/3520#discussion_r840458252